src/adapters/picked_team_adapters.py

```diff
@@ -0,0 +1,9 @@
+from core.picked_team import PickedTeam
+
+
+def to_picked_team(json: dict) -> PickedTeam:
+    return PickedTeam(
+        name=json["picked_team"]["name"],
+        score=json["picked_team"]["score"],
+        picked_players=json["picked_team"]["picked_players"],
```

Comment:
According to PickedTeam __init__ method, I would use something like this:

```suggestion
        picked_players=PickedPlayer(<params here>)
```